packages/fast-usdc/src/exos/advancer.js

```diff
@@ -1,5 +1,7 @@
+import { AmountMath, AmountShape, BrandShape } from '@agoric/ertp';
 import { assertAllDefined } from '@agoric/internal';
 import { ChainAddressShape } from '@agoric/orchestration';
+import { pickFacet } from '@agoric/vat-data';
```

Comment:
out of scope, but this helper should move eventually. It's about Exos, not vat data.

We could move it down to `@agoric/internal` but that is meant to not be consumed externally and this is valuable to other contract authors. 

 I wonder if it belongs in Endo.